src/LibVLCSharp.Avalonia.Unofficial/README.md

```diff
@@ -0,0 +1,15 @@
+# LibVLCSharp.Avalonia.Unofficial
```

Comment:
Thanks for these notes. They could likely be added to https://github.com/videolan/libvlcsharp/blob/3.x/src/LibVLCSharp.Avalonia/README.md